backend/Platform/settings/production.py

```diff
@@ -43,3 +43,14 @@
 EMAIL_USE_TLS = True
 
 IS_DEV_LOGIN = os.environ.get("DEV_LOGIN", "False") in ["True", "TRUE", "true"]
+
+# AWS S3
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+AWS_ACCESS_SECRET_ID = os.getenv("AWS_SECRET_ACCESS_KEY")
+AWS_STORAGE_BUCKET_NAME = "penn.platform"
+AWS_S3_SIGNATURE_VERSION = "s3v4"
```

Comment:
Is there any reason we need the configurations specified in these following lines? I don't see them being used in our other products that use S3